docs/api/en/objects/BatchedMesh.html

```diff
@@ -14,32 +14,42 @@ <h1>[name]</h1>
 		<p class="desc">
 			A special version of [page:Mesh] with multi draw batch rendering support. Use
 			[name] if you have to render a large number of objects with the same
-			material but with different world transformations and geometry. The usage
-			of [name] will help you to reduce the number of draw calls and thus
-			improve the overall rendering performance in your application.
+			material but with different world transformations. The usage of [name] will
+			help you to reduce the number of draw calls and thus improve the overall
+			rendering performance in your application.
 
 			<br/>
 			<br/>
 
 			If the [link:https://developer.mozilla.org/en-US/docs/Web/API/WEBGL_multi_draw WEBGL_multi_draw extension] is
-			not supported then a less performant callback is used.
+			not supported then a less performant fallback is used.
 		</p>
 
 		<h2>Code Example</h2>
 
 		<code>
 		const box = new THREE.BoxGeometry( 1, 1, 1 );
-		const sphere = new THREE.BoxGeometry( 1, 1, 1 );
+		const sphere = new THREE.SphereGeometry( 1, 12, 12 );
 		const material = new THREE.MeshBasicMaterial( { color: 0x00ff00 } );
 
 		// initialize and add geometries into the batched mesh
 		const batchedMesh = new BatchedMesh( 10, 5000, 10000, material );
-		const boxId = batchedMesh.addGeometry( box );
-		const sphereId = batchedMesh.addGeometry( sphere );
+		const boxGeometryId = batchedMesh.addGeometry( box );
+		const sphereGeometryId = batchedMesh.addGeometry( sphere );
+
+		// create instances of those geometries
+		const boxInstancedId1 = batchedMesh.addInstance( boxGeometryId ); 
+		const boxInstancedId2 = batchedMesh.addInstance( boxGeometryId );
+
+		const sphereInstancedId1 = batchedMesh.addInstance( sphereGeometryId ); 
+		const sphereInstancedId2 = batchedMesh.addInstance( sphereGeometryId ); 
```

Comment:
Just an idea, certainly not necessary in this release. We could consider having `addInstance` take a Matrix4 as a second argument, so that users creating static geometry do not need to explicitly save the instance ID and call `setMatrixAt` later.